src/graph/backend/dnnl/subgraph.cpp

```diff
@@ -443,6 +443,26 @@ void subgraph_rewriter_t::insert_op_before(const op_ptr &inserted_op,
     auto in_dtype = in_val->get_logical_tensor().data_type;
     new_val->set_data_type(in_dtype);
 
+    if (inserted_op->get_kind() == op_kind::dnnl_permute
+            && (base_op->get_kind() == op_kind::dnnl_mul_scales
+                    || base_op->get_kind() == op_kind::dnnl_sub_zps)) {
+        // as DNNL backend only respect abx tag for scale and zps, should set
+        // strides explicitly and execute reorder.
```

Comment:
Nit: seems not the `DNNL backend` (maybe primitive kernel ??) respect abx tag. More clear clarification for this special case helps for later revisitor. Thanks~